www/src/public/js/controllers/builder/main.js

```diff
@@ -146,27 +147,34 @@
             // load lists
             if ($cookies.get("cookie-consent")) {
                 var listVersion = -1;
-                var listCookieStr = localStorage.getItem("cl2");
-                if (listCookieStr) {
-                    listVersion = 2;
-                }
-                else {
-                    var listCookieStr = localStorage.getItem("cl");
-                    if (!listCookieStr) {
-                        listCookieStr = $cookies.get("cl1");
-                    }
+                var listCookieStr = localStorage.getItem("cl3")
+                
+                // version check
+                if (localStorage.getItem("cl1")) {
+                    listCookieStr = localStorage.getItem("cl1");
                     listVersion = 1;
+                } 
+                if (localStorage.getItem("cl2")) {
+                    listCookieStr = localStorage.getItem("cl2");
+                    listVersion = 2;
                 }
+                if (localStorage.getItem("cl3")) {
+                    listCookieStr = localStorage.getItem("cl3");
+                    listVersion = 3;
+                }                    
 
                 if (listCookieStr) {
                     var listStrs = listCookieStr.split("*").filter(function(el) {return el.length != 0});
                     for (var i = 0; i < listStrs.length; ++i) {
                         switch (listVersion) {
-                            case 1:
-                                var newList = createListFromString(listStrs[i]);
+                            case 3:
```

Comment:
since case 3 and 2 are the same you can combine them into one case:

```
switch (listVersion) {
  case 1:
    var newList = createListFromString(listStrs[i]);
    break;
  case 2:
  case 3:
    var newList = createListFromStringV2(listStrs[i], listVersion);
    break;
  default:
    break;
}
```

Reply:
If factorio has taught me anything its that optimization and efficiency are key.  Fixed.